application/controllers/admin/tokens.php

```diff
@@ -153,13 +149,13 @@ public function bounceprocessing($iSurveyId)
 
             switch ($hostencryption) // novalidate-cert to have personal CA , maybe option.
```

Comment:
Because you switched to lowercase settings you must make sure that you convert the setting to lowercase first before comparing here.

Reply:
$hostencryption is converted on Line 102 and Line 108 above. The same lines which were used to convert to uppercase. I didn't change the logic.